src/EventCell.js

```diff
@@ -49,6 +50,10 @@ class EventCell extends React.Component {
     if (eventPropGetter)
       var { style, className: xClassName } = eventPropGetter(event, start, end, selected);
 
+    if (this.props.resizable) {
```

Comment:
These bits should be encapsulated in the addon code, the core libra should require things from addons (so that they are optional)

Reply:
ping

Reply:
I see what you mean here. In our implementation at work I ended up removing this.